### PR TITLE
Update README.zh-CN.md

### PR DESCRIPTION
**Pull request by Alexei007**

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -257,12 +257,12 @@ CSS Text Module Level 4 的 [`text-spacing`](https://www.w3.org/TR/css-text-4/#t
...
-| [Apple 台湾](https://www.apple.com/tw/)            | 是  | N/A  |
-| [Microsoft 中国](https://www.microsoft.com/zh-cn/) | 是  | N/A  |
-| [Microsoft 香港](https://www.microsoft.com/zh-hk/) | 是  | N/A  |
-| [Microsoft 台湾](https://www.microsoft.com/zh-tw/) | 是  | N/A  |
+| [Apple 中国大陆](https://www.apple.com/cn/)            | 是  | N/A  |
+| [Apple 中国香港](https://www.apple.com/hk/)            | 是  | N/A  |
+| [Apple 中国台湾](https://www.apple.com/tw/)            | 是  | N/A  |
+| [Microsoft 中国大陆](https://www.microsoft.com/zh-cn/) | 是  | N/A  |
+| [Microsoft 中国香港](https://www.microsoft.com/zh-hk/) | 是  | N/A  |
+| [Microsoft 中国台湾](https://www.microsoft.com/zh-tw/) | 是  | N/A  |
```
Comment (liyishuai) [inline README.zh-CN.md:265]:
```suggestion
| [Apple (中国大陆)](https://www.apple.com/cn/)            | 是  | N/A  |
| [Apple (香港)](https://www.apple.com/hk/)            | 是  | N/A  |
| [Apple (台湾)](https://www.apple.com/tw/)            | 是  | N/A  |
| [Microsoft 中文(中国)](https://www.microsoft.com/zh-cn/) | 是  | N/A  |
| [Microsoft 中文(香港特别行政区)](https://www.microsoft.com/zh-hk/) | 是  | N/A  |
| [Microsoft 中文(台湾)](https://www.microsoft.com/zh-tw/) | 是  | N/A  |
```
Align with the website contents verbatim.

Comment (Alexei007) [reply README.zh-CN.md:265]: Hong Kong and Taiwan are not countries. This is common sense, not politics. Is correction disrespectful?

Comment (sgalal) [reply README.zh-CN.md:265]: > 
> 
> Hong Kong and Taiwan are not countries. This is common sense, not politics. Is correction disrespectful?

「国家和地区」

Comment (liyishuai) [reply README.zh-CN.md:265]: https://www.apple.com/choose-country-region/
> Choose your country or region
> - 中国大陆
> - Hong Kong (English)
> - 香港
> - 澳門
> - 台灣

https://www.microsoft.com/zh-cn/locale.aspx
> 本网站有其他国家/地区版本：
> - 中国 - 简体中文
> - 台灣 - 繁體中文
> - 香港特別行政區 - 繁體中文

I don't see a problem with these phrasings. That's why I suggested using the country/region names as shown on the website verbatim.

Comment (liyishuai) [reply README.zh-CN.md:265]: If you insist on being politically correct, then `香港` should be in parallel with `内地`, not `大陆`.

**Pull request closed without merging**